cli/src/commands/serve.ts

```diff
@@ -0,0 +1,52 @@
+import { Command, flags } from "@oclif/command";
+import { outputFile } from "../../../lib/src/io/output";
+import { parse } from "../../../lib/src/neu/parser";
+import { runValidationServer } from "../../../lib/src/validation-server/server";
+
+const ARG_API = "spot_contract";
+
+/**
+ * oclif command to start the spot contract validation server
+ */
+export default class Serve extends Command {
+  static description = "Start the spot contract validation server";
+
+  static examples = ["$ spot serve api.ts"];
+
+  static args = [
+    {
+      name: ARG_API,
+      required: true,
+      description: "path to Spot contract",
+      hidden: false
+    }
+  ];
+
+  static flags = {
+    help: flags.help({ char: "h" }),
+    port: flags.integer({
+      char: "p",
+      default: 5907,
+      description: "The port where application will be available"
+    })
+  };
+
+  async run() {
+    const { args, flags } = this.parse(Serve);
+    const contractPath = args[ARG_API];
+    const { port } = flags;
+
+    try {
+      this.log("Generating raw contract...");
+      const rawContract = JSON.stringify(parse(contractPath));
+      this.debug("Outputing raw contract to ./spot");
+      outputFile(".spot", "raw.json", rawContract, true);
```

Comment:
Why do we still need to generate the raw file?

Reply:
Now that you said that, I see we could just use the contract in memory instead, that's what you were thinking?